workflow/scripts/build_existing_chp_de.py

```diff
@@ -16,20 +16,30 @@
 import sys
 
 
-def clean_data(combustion, biomass, geodata):
+def clean_data(combustion, biomass, geodata, opts):
     """
     Clean the data and return a dataframe with the relevant information.
     PLZ is translated to longitude and latitude using the pyGeoDb data.
     """
     biomass.dropna(subset="Postleitzahl", inplace=True)
     biomass.rename(columns={'NameStromerzeugungseinheit': 'NameKraftwerk'}, inplace=True)
 
+    if 'I' not in opts:
+        # filter out industrial CHPs if industry is not included
+        industry = combustion.dropna(subset='Einsatzort')
+        to_drop = industry[industry.Einsatzort.str.contains('Industrie')].index
+        combustion.drop(index=to_drop, inplace=True)
```

Comment:
I think this could be simplified to:

```py
industry_b = combustion.Einsatzort.fillna("").str.contains("Industrie")
combustion = combustion.loc[~industry_b].copy()
```